nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/main/java/org/apache/nifi/provenance/lucene/DocsReader.java

```diff
@@ -47,9 +46,6 @@
 public class DocsReader {
     private final Logger logger = LoggerFactory.getLogger(DocsReader.class);
 
-    public DocsReader(final List<File> storageDirectories) {
-    }
-
```

Comment:
We have designed flexibility in for a reason.  Deprecation is appropriate for the public portions of code designed for extension and/or modification.  For everything else let's be sure it is clear and documented what is fair game for the community to evolve.  If it isn't clear then we should err on the side of deprecation then make it clear.  We need to have agility behind the curtain.